apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -760,8 +1095,11 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     # the default value below is true, so that the assumption is that utxo not checked is **present**
     # TODO: rather inefficient, but no as inefficient as the nested `filter` calls in searching for competitors
     #       consider optimizing using `MapSet`
+
     utxo_positions
-    |> Enum.filter(fn pos -> !Map.get(utxo_exists?, pos, true) end)
+    |> Enum.filter(fn utxo_pos ->
```

Comment:
are you attached into this multi-line version of this :)?

Reply:
done